Prepare for release (#54)
0xPolygonZero/zk_evm: docs/usage_seq_diagrams.md

# Usage Diagrams

> These are some diagrams showing how the protocol will (likely) be implemented.

Reviewer: ```suggestion
These are some diagrams showing how the protocol is implemented.
```
